routers/routes/routes.go

```diff
@@ -710,6 +710,10 @@ func RegisterRoutes(m *macaron.Macaron) {
 		m.Post("/purge", user.NotificationPurgePost)
 	}, reqSignIn)
 
+	if setting.API.EnableSwaggerEndpoint {
```

Comment:
You should add to https://github.com/go-gitea/gitea/blob/master/custom/conf/app.ini.sample#L556 that it also disable the json.

Reply:
It was added in #3502.